test/test_jit_tracing.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+from common import TEST_DEVICES
+
+import torch
+import torchgeometry as tgm
+import torch.nn as nn
+
+
+class WarpPerspective(nn.Module):
+
+    def __init__(self, output_size):
+        super(WarpPerspective, self).__init__()
+
+        self.output_height, self.output_width = output_size
+
+    def forward(self, x, M):
+
+        x = tgm.warp_perspective(x, M,
+                                 dsize=(self.output_height,
+                                        self.output_width))
+
+        return x
+
+
+class WarpAffine(nn.Module):
+
+    def __init__(self, output_size):
+        super(WarpAffine, self).__init__()
+
+        self.output_height, self.output_width = output_size
+
+    def forward(self, x, M):
+
+        x = tgm.warp_affine(x, M,
+                            dsize=(self.output_height,
+                                   self.output_width))
+
+        return x
+
+
+class LocalizationNetwork(nn.Module):
+
+    def __init__(self, output_size, pretrained=True, affine=True):
+        super(LocalizationNetwork, self).__init__()
+
+        self.cnn = nn.Sequential(
```

Comment:
do we really need that many layers ? could just work let's say one block of conv+relu+max-pool with very few neurons (e.g. 1-3) ?

Reply:
I have just simplified the model.